docs/QuickstartDeployPortal.md

```diff
@@ -28,20 +28,25 @@ Below is a table that describes all of the parameters in the custom deployment t
 |Service Name|**Yes**|Name of the FHIR Service Web App (i.e. MyFHIRServer)|
 |App Service Plan Resource Group|No|Specify the exact name of your App Service Plan Resource where your App Service Plan lives or leave blank if you want to create a new one|
 |App Service Plan Name|No|If you leave this blank, a new one will be created. If you want to use an existing one, you will need to specify the name exactly. Note that an App Service Plan can be either Linux or Windows. The deployment from this page creates a Windows App and so you will need a Windows App Service Plan. If you use the [Docker deployment](https://github.com/microsoft/fhir-server/blob/master/samples/templates/default-azuredeploy-docker.json) you will need a Linux App Service Plan|
+|App Service Plan Sku|No|The Sku of App Service Plan to use if a new one is created.
```

Comment:
```suggestion
|App Service Plan Sku|No|The Sku of App Service Plan to use. If blank, a new plan is created.
```

Reply:
This parameter doesn't control if a new app service plan is made, App Service Plan Name controls that. This just controls which sku is used if a new plan is made.

Reply:
So something along the lines of "Specifies the name of a new or existing app service plan"?